### PR TITLE
TE-261 Select Test execution environment from Testproject editor

### PR DESCRIPTION
**Pull request by kpanier**
- Add new fixture lib to create directories in workspace
- Add id to combo box for test lookup
- Rename LOGGER to logger

Task-Url: https://testeditor.atlassian.net/browse/TE-261

### CODE REVIEW

File: ui/org.testeditor.ui/src/main/java/org/testeditor/ui/parts/projecteditor/TestProjectEditor.java [hunk 1]
```diff
@@ -521,7 +523,7 @@ protected void refreshAfterRevert(@UIEventTopic(TestEditorUIEventConstants.TESTS
 				testProjectService.reloadTestProjectFromFileSystem(getTestProject());
 				setTestProject(getTestProject());
 			} catch (SystemException e) {
-				LOGGER.error(e.getMessage());
+				logger.error(e.getMessage());
```
Comment (franzbecker) [inline ui/org.testeditor.ui/src/main/java/org/testeditor/ui/parts/projecteditor/TestProjectEditor.java]: Why is the `SystemException` handled differently here?

**Comment by kpanier** [comment]
It doesn't make sense to handle the exception that way. I changed it.

**Comment by franzbecker** [comment]
Thanks, but the exception handling in `save` still differs from the one in `refreshAfterRevert`.
Does it make sense to extract this logic into a helper method, e.g. `handleSystemException(String, SystemException`?

```
private void handleSystemException(String performedAction, SystemException e) {
    logger.error("Error while performing " + performedAction, e);
    // ...
}
```

Call could look like this:

```
} catch (SystemException e) {
    handleSystemException("refresh after revert", e);
}
```

**Comment by kpanier** [comment]
The save operation is called from a non ui thread, so that we have to create a new runnable to open the error message.
If we don't have the same thread situation in the refreshAfterRevert called by the event broker, i prefer the simpler solution. 
If wan't the same solution for all exception in the ui classes, we should build a utility class which is created for every exception handling in the way of the save operation. But that should be a new refactoring user story.

**Pull request merged**